Makefile

```diff
@@ -74,7 +74,7 @@ generate-llhttp: .llhttp-gen
 cythonize: .install-cython $(PYXS:.pyx=.c)
 
 .install-deps: .install-cython $(PYXS:.pyx=.c) $(call to-hash,$(CYS) $(REQS))
-	@pip install -r requirements/dev.txt -c requirements/constraints.txt
+	@pip install -r requirements/dev.txt -c requirements/constraints.txt --user
```

Comment:
This probably won't work for the contributors as they usually run under venvs and using it with error out for them. I would probably try using either an env var or the config file just in the CI.